eo-runtime/src/main/java/org/eolang/AtLogged.java

```diff
@@ -44,14 +44,30 @@ final class AtLogged implements Attr {
      */
     private final String owner;
 
+    /**
+     * Output stream.
+     */
+    private final PrintStream out;
```

Comment:
@volodya-lombrozo why don't we use a logger here like we do in `PhDefault` and `Dataized`?

Reply:
@mximp Yea, I agree, it would be much better. I've added puzzle for that. Check, please.